dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/processor/queue/TaskResponseEvent.java

```diff
@@ -128,6 +133,7 @@ public static TaskResponseEvent newResult(ExecutionStatus state,
         event.setEvent(Event.RESULT);
         event.setVarPool(varPool);
         event.setChannel(channel);
+        event.setResult(result);
```

Comment:
When sending an ack message, do you really need this part of the code?

Reply:
not big problem